external/external.go

```diff
@@ -32,6 +32,17 @@ func (eh *ExternalHandler) EmailSubscription(w http.ResponseWriter, r *http.Requ
 	SubDoc, _ := utils.GetMongoDbDoc(newsletter_collection, bson.M{"email": NewSubscription.Email})
 	if SubDoc != nil {
 		// fmt.Printf("user with email %s already subscribed!", NewSubscription.Email)
+		msger := eh.mailService.NewMail(
+			[]string{NewSubscription.Email}, "Zuri Chat Newsletter Subscription", service.EmailSubscription,
+			&service.MailData{
+				Username: NewSubscription.Email,
+				ZuriLogo: utils.ConvertImageTo64("./templates/email_sub/images/zuri_logo.png"),
+				Image2:   utils.ConvertImageTo64("./templates/email_sub/images/people_chatting.png"),
```

Comment:
Awesome job @utukJ 
But can you convert the image online and copy the base64 code on the html file. Can you do that now, so we can merge.